bin/serve.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../scripts/serve/serve-cli');
+require('webpack-dev-server/bin/webpack-dev-server');
```

Comment:
The file permissions on `bin/serve.js` and `bin/serve-static.js` changed

Reply:
This was on purpose to make the bin files executable locally.

Reply:
Ah so are still good when releases as well? I just know once had bad file permissions and we had to do another release